core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -174,13 +174,13 @@ ifeq ($(CFG_WITH_PAGER),y)
 CFG_TEE_CORE_DEBUG ?= n
 CFG_UNWIND ?= n
 CFG_LOCKDEP ?= n
-CFG_CORE_ASLR ?= n
 CFG_TA_BGET_TEST ?= n
 # Default disable early TA compression to support a smaller HEAP size
 CFG_EARLY_TA_COMPRESS ?= n
 CFG_CORE_HEAP_SIZE ?= 49152
 endif
 
+CFG_CORE_ASLR ?= n
```

Comment:
nitpicking: for consistency of _conf.mk_ layout, I would prefer this line is moved next to generic configs, around lines 46/50:
```patch
 CFG_WITH_LPAE ?= y
 CFG_MMAP_REGIONS ?= 23
 CFG_DTB_MAX_SIZE ?= (256 * 1024)
+CFG_CORE_ASLR ?= n
 
 ifeq ($(CFG_EMBED_DTB_SOURCE_FILE),)
 # Some drivers mandate DT support
```